src/styles/embeds/sass/components/product.css

```diff
@@ -305,16 +305,33 @@
 
 .carousel-button {
   position: absolute;
-  top: 50%;
-  transform: translateY(-50%);
+  width: 75px;
+  top: 0;
+  height: 100%;
+  border: none;
+  font-size: 0;
+  background-image: url("http://10.91.1.169:8080/dist/arrow.svg");
```

Comment:
let's update this to what it will be once uploaded (once you're otherwise ready to merge)